great_ape_safe/ape_api/citadel.py

```diff
@@ -8,59 +8,73 @@
 Limits = namedtuple("limits", "min max")
 
 
-class Citadel():
+class Citadel:
     """
     Collection of all contracts and methods needed to interact with the Citadel
     system.
     """
 
-
     def __init__(self, safe):
         self.safe = safe
 
+        # constants
+        self.MAX_BPS = 10000
         # contracts
         self.gac = interface.IGlobalAccessControl(
             registry.eth.citadel.gac, owner=self.safe.account
         )
-
+        self.citadel_minter = interface.ICitadelMinter(
+            registry.eth.citadel.minter, owner=self.safe.account
+        )
 
     def get_funding_contract(self, asset):
         return interface.IFunding(
-            registry.eth.citadel.funding[asset],
-            owner=self.safe.account
+            registry.eth.citadel.funding[asset], owner=self.safe.account
         )
 
-
     def get_discount(self, asset):
-        return self.get_funding_contract(asset).getDiscount();
-
+        return self.get_funding_contract(asset).getDiscount()
 
     def set_discount(self, asset, discount):
         contract = self.get_funding_contract(asset)
         contract.setDiscount(discount)
         assert discount == self.get_discount(asset)
 
-
     def get_discount_limits(self, asset):
         contract = self.get_funding_contract(asset)
-        return Limits(
-            contract.getFundingParams()[1],
-            contract.getFundingParams()[2]
-        )
```

Comment:
@petrovska-petro the reason these are not on one line is because the PEP8 style guide dictates a max line length of 79: https://peps.python.org/pep-0008/#maximum-line-length

by bringing them back to one line you are making it exceed this length: <img width="800" alt="Screenshot 2022-05-05 at 14 20 39" src="https://user-images.githubusercontent.com/2835259/166921678-8951b943-84c0-469c-931a-62834e1084f5.png">

Reply:
discussed on DM, we can enforce it at the repository level